addon/controllers/project/form.js

```diff
@@ -320,4 +320,25 @@ export default class ProjectFormController extends ImportController {
   get workWithBuildings() {
     return this.project.work.filter((work) => !work.building.isNew);
   }
+
+  @action
+  isClientMismatch(changeset) {
+    const personNamePath =
+      "client.identification.personIdentification.officialName";
+    const organisationNamePath =
+      "client.identification.organisationIdentification.organisationName";
+
+    const isSet = (value) =>
+      value !== undefined && value !== null && value !== "";
```

Comment:
Is the explicit check for the three values necessary or can we just check if it is false?